frontend/src/Wrapper.tsx

```diff
@@ -105,6 +127,14 @@ export default function Wrapper({ children }: WrapperProps): ReactElement {
               </>
             )}
           </Box>
+          {isUiConfigLoading && <Loading />}
+          {uiConfig && (
+            <Box sx={{ mb: 2 }}>
```

Comment:
Is this box necessary? I don't think we want the banner to take up any space from the existing content, we want it to sit on top of it.

Also consider combining the two conditional blocks into:
```
{uiConfig && annoucementBannerOpen && ( ... )}
```